README.md

```diff
@@ -229,16 +229,126 @@ Pre-requisites:
 * (Recommended) You have a strategy for where to publish your Controller
   Documents and Public Keys
 
-To create a verifiable presentation out of one or more verifiable credentials:
+#### Creating an Unsigned Presentation
+
+To create a presentation out of one or more verifiable credentials, you can
+use the `createPresentation()` convenience function. Alternatively, you can
+create the presentation object manually (don't forget to set the `@context` and
+`type` properties).
+
+To create a verifiable presentation with a custom `@context` field use a [custom documentLoader](#custom-documentLoader)
 
 ```js
-const verifiableCredential = [vc1, vc2];
-const vp = await vc.createPresentation({verifiableCredential, suite});
-console.log(JSON.stringify(vp, null, 2));
+const verifiableCredential = [vc1, vc2]; // either array or single object
+
+// optional `id` and `holder`
+const id = 'ebc6f1c2';
+const holder = 'did:ex:12345';
+
+const presentation = vc.createPresentation({
+  verifiableCredential, id, holder
+});
+
+console.log(JSON.stringify(presentation, null, 2));
+// ->
+{
+  "@context": [
+    "https://www.w3.org/2018/credentials/v1"
+  ],
+  "type": [
+    "VerifiablePresentation"
+  ],
+  "id": "ebc6f1c2",
+  "holder": "did:ex:12345",
+  "verifiableCredential": [
+    // vc1:
+    {
+      "@context": [
+        "https://www.w3.org/2018/credentials/v1",
+        "https://www.w3.org/2018/credentials/examples/v1"
+      ],
+      "id": "http://example.edu/credentials/1872",
+      "type": [
+        "VerifiableCredential",
+        "AlumniCredential"
+      ],
+      "issuer": "https://example.edu/issuers/565049",
+      "issuanceDate": "2010-01-01T19:23:24Z",
+      "credentialSubject": {
+        "id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+        "alumniOf": "<span lang=\"en\">Example University</span>"
+      },
+      "proof": {
+        "type": "Ed25519Signature2018",
+        "created": "2020-02-03T17:23:49Z",
+        "jws": "eyJhbGciOiJFZERTQSIsImI2NCI6ZmFsc2UsImNyaXQiOlsiYjY0Il19..AUQ3AJ23WM5vMOWNtYKuqZBekRAOUibOMH9XuvOd39my1sO-X9R4QyAXLD2ospssLvIuwmQVhJa-F0xMOnkvBg",
+        "proofPurpose": "assertionMethod",
+        "verificationMethod": "https://example.edu/issuers/keys/1"
+      }
+    },
+    // vc2 goes here ...
+  ]
+}
 ```
 
-To create a verifiable presentation with a custom `@context` field use a [custom documentLoader](#custom-documentLoader)
+Note that this creates an _unsigned_ presentation (which may be valid
```

Comment:
@dlongley I believe unsigned presentations should be the norm? @dmitrizagidulin I could be wrong on this one.

Reply:
That is correct, yes.